src/lando/api/legacy/workers/landing_worker.py

```diff
@@ -143,6 +144,15 @@ def run_job(self, job: LandingJob) -> bool:
 
         job.transition_status(JobAction.LAND, commit_id=commit_id)
 
+        if job.is_pull_request_job:
+            # TODO: move this to different method, and retry if needed.
+            # NOTE: This may need to happen on the revision-level when stack support is added.
+            pull_number = job.revisions.first().pull_number
+            message = f"Pull request closed by commit {commit_id}"
+            client = GitHubAPIClient(job.target_repo)
```

Comment:
I suspect this is what makes the tests fail, though not sure why as non-GitHub repos as used in tests should not have `pull_number` on their revision. Unless a default is wrong, perhaps?

Reply:
> Unless a default is wrong, perhaps?

Nope.

https://github.com/mozilla-conduit/lando/blob/1c63a7a08057a37e07f54b60a60f1e88ff356669/src/lando/main/models/revision.py#L50